src/table/handle-scroll.js

```diff
@@ -0,0 +1,13 @@
+const handleScroll = (evt, tableSection) => {
+  evt.stopPropagation();
+  const max = tableSection.current.scrollWidth - tableSection.current.offsetWidth;
+  if (
+    max > 0 &&
+    (tableSection.current.scrollLeft + evt.deltaX < 0 || tableSection.current.scrollLeft + evt.deltaX > max)
```

Comment:
`tableSection.current.scrollLeft` is used 3 times, save it as a constant

Reply:
sure